Documentation/ceph-upgrade.md

```diff
@@ -109,21 +118,27 @@ created before v0.8 are likely to be:
 * Clusters created in v0.8 or v0.9: `rook-ceph-system` and `rook-ceph`
 * Clusters created in v1.0 or newer: only `rook-ceph`
 
-With this guide, we do our best not to assume the namespaces in your cluster.
-To make things as easy as possible, modify and use the below snippet to configure your environment.
-We will use these environment variables throughout this document.
+With this guide, we do our best not to assume the namespaces in your cluster. To make things as easy
+as possible, modify and use the below snippet to configure your environment. We will use these
+environment variables throughout this document.
 
 ```sh
 # Parameterize the environment
 export ROOK_SYSTEM_NAMESPACE="rook-ceph"
 export ROOK_NAMESPACE="rook-ceph"
 ```
 
+You must also perform a sed-replace on one of the upgrade manifests to set the namespace for your
+unique cluster.
+```sh
+sed -i.bak "s/__ROOK_CEPH_CLUSTER_NAMESPACE__/$ROOK_NAMESPACE/g" upgrade-from-v1.2-apply.yaml
```

Comment:
I'll change it, but this is what we did for a release or two back, so I was just following that trend.

Reply:
Yeah I think we had to do it before since there were other `namespace` elements that couldn't be replaced, or applied differently for the system namespace and regular rook namespace. Looks like we don't have that conflict this time.